doc: event_manager_proxy: Document the library limitations (#7632)
nrfconnect/sdk-nrf: doc/nrf/libraries/others/event_manager_proxy.rst

After the link is initialized, the subscribed events from the remote core appear in the local event queue.
The events are then processed like any other local messages.

Limitations
***********

The event passed through the Event Manager Proxy is treated and processed the same way that any locally generated event.

> The core that sources the event should not subscribe for the same event in another core.

Author: I am not sure if it must not - In some cases we wish the core to resend the event.

If it does, after it receives such an event generated remotely, it would resend it automatically from that local core to the ones that subscribed to it.

> We would run into really dangerous sitiation when two cores subscribe to the same event in each other.

Reviewer: ```suggestion
We would run into a dangerous situation where two cores subscribe to the same event.
```
